src/ui/layout/BrowseBar.vue

```diff
@@ -362,6 +362,10 @@ export default {
         title: 'Saving'
       });
 
+      const currentSelection = this.openmct.selection.selected[0];
+      const parentObject = currentSelection[currentSelection.length - 1];
+      this.openmct.selection.select(parentObject);
```

Comment:
Added to reset selection of views when they're saved.

Reply:
Isn't the desired behavior to stay on the saved object?

Reply:
ok. thanks for explaining this is for when you edit objects in display layouts, you should select the layout after saving. that makes sense.

If you aren't in a display layout, the code will select itself as the parent, and selection api takes care of selecting the same selection so things aren't getting re-mounted.